app/routes/recommend-to-pay.js

```diff
@@ -16,6 +19,21 @@ module.exports = {
       }
     },
     handler: async (request, h) => {
+      const userName = getUser(request).username
+      const response = await addStageExecution({
+        applicationReference: request.payload.reference,
+        stageConfigurationId: 1,
+        executedAt: new Date(),
+        executedBy: userName,
+        action: {
+          action: 'Recommend to pay'
```

Comment:
Minor - can we have these in a constants file?

Reply:
Good spot - they already are, I'll import it now